index.html

```diff
@@ -0,0 +1,95 @@
+                    type="text"
+                    placeholder="Name"
+                    id="name"        
+                    >
+                </div>
+
+                <div class= "divsInCommon" id="secondInput"> 
+                    <label id="email-label">Email</label>
+                    <input name="email"
+                    class="formInput"
+                    type="email"
+                    placeholder="Email"
+                    id="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" required
+                    >
+                </div>
+
+                <div class= "divsInCommon" id="thirdInput"> 
+                    <label id="number-label">Age</label>
+                    <input name="number"
+                    class="formInput"
+                    type="number"
+                    placeholder="Age"
+                    id="number" pattern="[0-9]" min="18" max="100" required
+                    >
+                </div>
+
+                <div class= "divsInCommon" id="fourthInput"> 
+                    <label id="dropdown" for="roles">Current Role</label>
+                    <br>
+                    <select name="roles" class="formInput" id="optionsInput">
+                        <option value="role1">
+                            Option 1
+                        </option>
+                        <option value="role2">
+                            Option 2
+                        </option>
+                        <option value="role3" selected>
+                            Option 3
+                        </option>
+                    </select>
+                </div>
+
+                <div class= "divsInCommon" id="fifthInput">
+                    <label id="recommend">Would you recommend our services?</label>
+                    <div>
+                        <input type="radio" class="inputRadio" name="recommend" value="2">
+                        <label for="definitely">Definitely!</label>
+                    </div>
+                    <div>
+                        <input type="radio" class="inputRadio" name="recommend" value="2">
+                        <label for="unsure">Unsure</label>
+                    </div>
+                    <div>
+                        <input type="radio" class="inputRadio" name="recommend" value="2">
+                        <label for="no">Totally not!</label>
+                    </div>
+                </div>
+
+                <div class="divsInCommon" id="sixthInput">
+                    <label id="suggestions">Suggestions? Comments?</label>
+                    <br>
+                    <textarea class="formInput" id="comment" placeholder="Share your suggestions here!"></textarea>
+                </div>
+
+                <div>
+                    <button type="submit" id="submit">
+                        Submit
+                    </button>
+                </div>
+            </form>
+        </div>
```

Comment:
I recommend you use other HTML semantic tags such as `<section>` rather than the much-used `<div>`'s, as well as using the `<a>` tag for buttons, instead of the nested `<button>` in the `<div>` element.

Reply:
Kindly explain why you requested this change, it might be obvious to you but not to the student 😄 👍

Reply:
@Danieldotcomcoder In some of the code reviews others have done on my own code, I have been asked for these changes so as to not clutter the code with too many div elements, and to use the preferred semantic tags as it is easier to read. Also serves for easier and more understandable code for both any other programmers who might want to modify it as well as anyone who is being read the page to them.